### PR TITLE
updated SS

### PR DESCRIPTION
**Pull request by Devashish-Thakar**
Changes made (required to update the execute file)

1. Removed w_init from function call

2. Removed convex combination of w since that depends on r

3. Removed printing of w because of 2

### CODE REVIEW

**Comment by jdebacker** [comment]
LGTM, merging.

**Comment by Devashish-Thakar** [comment]
Also updated the HH file to include bequests (BQ).

Will update the SS file to call in the get_BQ and summation over working age population

Thanks

File: scripts/households.py [hunk 1]
```diff
@@ -63,11 +63,11 @@ def FOCs(b_sp1, n_s, *args):
 
 
 
-def get_c(r, w, n_s, b_s, b_sp1):
+def get_c(r, w, n_s, b_s, b_sp1, BQ, N_tilte):
     '''
     Use the budget constraint to solve for consumption
     '''
-    c = w * n_s + (1 + r) * b_s - b_sp1
+    c = w * n_s + (1 + r) * b_s + (BQ / N_tilde) - b_sp1
```
Comment (jdebacker) [inline scripts/households.py]: You won't need `N_tilde` here.  See equation 8.29.  All the variables in this model will be the "hat" variables - these are the stationary versions of each variable.

**Comment by Devashish-Thakar** [comment]
Thanks, makes sense

**Comment by jdebacker** [comment]
Thanks for the changes @devashish2019, just merged.

**Pull request merged**